python/looker_sdk/rtl/transport.py

```diff
@@ -44,14 +44,23 @@ class HttpMethod(enum.Enum):
     HEAD = 7
 
 
+TConnectTimeout = int
+TReadTimeout = int
+TConnectAndReadTimeout = int
+TConnectAndReadTimeoutSeconds = Union[
+    Tuple[TConnectTimeout, TReadTimeout], TConnectAndReadTimeout
+]
+
+
 @attr.s(auto_attribs=True, kw_only=True)
 class TransportSettings:
     """Basic transport settings.
     """
 
-    base_url: str
+    base_url: str = ""
     api_version: str = "3.1"
     verify_ssl: bool = True
+    timeout: TConnectAndReadTimeout = (120, 120)
```

Comment:
```suggestion
    timeout: TConnectAndReadTimeoutSeconds = (120, 120)
```

Reply:
though we may want to rename `TConnectAndReadTimeoutSeconds` to something simpler like `TTimeoutSeconds`